sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -3504,7 +3504,7 @@ test_suite(
             gpr_copy_commands = gdb_commands_copy_registers(IBEX_GPRS),
             gpr_restore_commands = gdb_commands_restore_registers(IBEX_GPRS),
             gpr_set_commands = gdb_commands_set_registers("0x89abcdef", IBEX_GPRS),
-            watchdog_ctrl_reg_addr = "0x40470014",
+            watchdog_ctrl_reg_addr = "0x40470044",
```

Comment:
There are two hardcoded constants in this BUILD rule - any suggestions how we could use the autogenerated header constants here?